hdf5/src/lib.rs

```diff
@@ -206,6 +206,8 @@ pub fn is_library_threadsafe() -> bool {
     }
 }
 
+pub const HDF5_VERSION: hdf5_sys::Version = hdf5_sys::HDF5_VERSION;
```

Comment:
```suggestion
/// The version of the HDF5 library used for compilation.
pub const HDF5_VERSION: hdf5_sys::Version = hdf5_sys::HDF5_VERSION;
```

Reply:
This is inherited from the docs of `hdf5_sys::HDF5_VERSION`, including the version number

Reply:
Turns out it is not when built as a dependency, so added the comment. Thanks!